spec/services/applications/change_funded_place_spec.rb

```diff
@@ -92,6 +92,32 @@
             expect(service.errors.messages_for(:application)).to include("You must accept the application before attempting to change the '#/funded_place' setting.")
           end
         end
+
+        context "with declarations" do
+          before do
+            params.merge!(funded_place: false)
+          end
+
+          %w[submitted eligible payable paid].each do |applicable_state|
```

Comment:
I think we should put constants in the `Declaration` model for billable and unbillable states; we reference these all the time and its a mess in ECF, there's no structure around the declaration state groups

Reply:
Thank you @ethax-ross. [Great call](https://github.com/DFE-Digital/npq-registration/pull/1435/commits/9255d42e3b2397a1250eb2cfd4c5b62878a36d48) to use `billable` states if this will be of extended use also in `NPQ`. Following the spirit of your comment [I have also extracted the scope](https://github.com/DFE-Digital/npq-registration/pull/1435/commits/843423eeaac28fdb9ef874de4fac68540371e1ba): `billable` which enables us to:

1. Align the name `billable` with the service
2. Increase the probability of not using the constants directly from our codebase (other than tests)

Any thoughts?

Reply:
Yep that sounds good thanks 👍